src/vue-testing-library.js

```diff
@@ -103,6 +103,42 @@ fireEvent.touch = async (elem) => {
   await fireEvent.blur(elem)
 }
 
+fireEvent.update = async (elem, value) => {
+  const tagName = elem.tagName
+  const type = elem.type
+
+  switch (tagName) {
+    case 'OPTION':
+      elem.selected = value
+
+      const parentElement = this.element.parentElement.tagName === 'OPTGROUP'
+        ? this.element.parentElement.parentElement
+        : this.element.parentElement
+
+      return fireEvent.change(parentElement)
+
+    case 'INPUT':
+      if (type === 'checkbox') {
+        elem.checked = value
+        return fireEvent.change(elem)
+      } else if (type === 'radio') {
+        elem.selected = value
+        return fireEvent.change(elem)
+      } else {
+        elem.value = value
+        return fireEvent.input(elem)
+      }
+
+    case 'TEXTAREA':
+      elem.value = value
+      return fireEvent.input(elem)
+
+    case 'SELECT':
+      elem.value = value
+      await fireEvent.change(elem)
```

Comment:
quick question! Should this line `return`, instead of `await`? If not, why does it need to be different?

Reply:
I missed that, updated now.

WRT change and input, I hadn't considered that update may replace them all. I would rather expose them and allow the user the choice of which to use than deprecate them. What do you think?

Reply:
This actually leads me to another question... 

I feel that just writing `.input` or `.change`, like the native events, makes a lot of sense. And it would be quite weird if `fireEvent.input` only worked in certain cases (when v-model is not used).

So... what if `input` and `change` would handle v-model properly, instead of creating a new event? Would it be hard?

Reply:
We could mask input or change, however we're still moving away from what is the idiomatic usage and taking away options from the end user (to use input or change as they are natively).

I'm not sure if there is an ideal solution here tbh, although I think update as it is here is probably the most flexible solution.

Reply:
You are right. Also adding a method is a non breaking change, while modifying the implementation of `.input` and `.change` might be. Let's add the update method and see how it goes!